charts/plane-ce/templates/workloads/beat-worker.deployment.yaml

```
@@ -46,4 +46,44 @@ spec:
+  labels:
+    app.name: {{ .Release.Namespace }}-{{ .Release.Name }}-beat-worker-hpa
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ .Release.Name }}-beat-worker-wl
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)